frontend/src/components/input/ButtonConected.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from "react";
+import { Link } from "react-router-dom";
+import { AuthContext } from "../../../context/AuthContext";
+
+function ButtonConected() {
+  const { connected } = useContext(AuthContext);
+  const reload = () => {
```

Comment:
faire une requête vers une route "/logout"
côté controller, faire une fonction logout() qui supprime le cookie :
  res.clearCookie("auth");

Reply:
that it